src/pages/what-is-ebpf.js

```diff
@@ -94,6 +94,14 @@ class Description extends React.Component {
             becoming contributors to major eBPF projects, eBPF.io will help you
             along the way.
           </p>
+          <h3>What is the bee named?</h3>
+          <p>
+            The bee is the official logo for eBPF. At the first 
+            <a href="https://ebpf.io/summit-2020">eBPF Summit</a>
+            there was a vote taken and the bee was named **eBee**.
+            (For details on acceptable uses of the logo, please see the Linux Foundation guidelines
+            <a href="https://linuxfoundation.org/brand-guidelines/">Brand Guidelines</a>.)
```

Comment:
```suggestion
            The bee is the official logo for eBPF. At the first  
            <a href="https://ebpf.io/summit-2020">eBPF Summit</a> 
            there was a vote taken and the bee was named **eBee**.
            (For details on acceptable uses of the logo, please see the Linux Foundation 
            <a href="https://linuxfoundation.org/brand-guidelines/">Brand Guidelines</a>.)
```

Reply:
I'm not sure of how this works under the hood, but it looks like spaces at the end or the beginning of the line are trimmed. Maybe try having the link on the same line as some other text, with no line space between e.g. “first” and `<a href=""...`